.circleci/config.yml

```diff
@@ -31,18 +36,21 @@ workflows:
 
 jobs:
 
-  build:
+  test_38: &test-template
     <<: *defaults
 
     steps:
       - checkout
 
-      # Download and cache dependencies
+      - run:
+          name: Prepare cache key
+          command: python --version > python_version.txt
+
       - restore_cache:
           keys:
-            - v5-dependencies-{{ checksum "Pipfile.lock" }}
+            - v6-dependencies-{{ checksum "python_version.txt" }}-{{ checksum "Pipfile.lock" }}
```

Comment:
Is "v6" for CircleCI or something else?

Reply:
This is how one does cache purging on CircleCI. You change the cache key.

In this case, since I added `python_version.txt` to the cache key, there is no need to bump the `V` number. Reverting.